azure/conf.py

```diff
@@ -71,9 +71,11 @@
         "../instances/create-pro-fips-golden-image/",
     "azure-how-to/upgrade-from-focal-to-jammy":
         "../instances/upgrade-from-focal-to-jammy/",
+    "azure-how-to/create-a-golden-image-pipeline":
+        "../instances/create-a-golden-image-pipeline/",
```

Comment:
This redirection is not needed and can be removed from here. These redirections are used for how-to guides that were written before the current subfolder structure was decided.